src/content/configuration/stats.md

```diff
@@ -146,6 +146,9 @@ module.exports = {
     // Show dependencies and origin of warnings/errors (since webpack 2.5.0)
     moduleTrace: true,
 
+    // Show outputPath
+    outputPath: true,
```

Comment:
is this true by default?

Reply:
@EugeneHlushko 
This default value decided by `forToString`

And I found this document not collected like chunks and source.

* `chunks: true`
  * https://github.com/webpack/webpack/blob/master/lib/Stats.js#L156
* `source: false`
  * https://github.com/webpack/webpack/blob/master/lib/Stats.js#L188

Which case should I suit?

Reply:
By reading the source code it is not obvious, i would suggest running stats and checking in debugger the default value, can you try that @terrierscript ?